modular_skyrat/modules/alerts/code/security_level_procs.dm

```diff
@@ -0,0 +1,11 @@
+/datum/controller/subsystem/security_level/proc/minimum_security_level(min_level = SEC_LEVEL_ORANGE, eng_access = TRUE, maint_access = FALSE)
+	var/sec_level = SSsecurity_level.get_current_level_as_number()
+	if(sec_level < min_level)
```

Comment:
```suggestion
	if(SSsecurity_level.get_current_level_as_number() < min_level)
```
You might not want to have this as a variable if it is not going to be used anywhere else inside of the proc